ts/package.ts

```diff
@@ -27,7 +25,7 @@ export function isPackage(x: unknown): x is Package {
  */
 export function mkPackage(
   nixExpression: NixExpression,
-  description?: string,
+  description: string,
```

Comment:
This is a breaking change of the typescript library. I do think that it's nicer to make the description be required.